packages/suite/src/actions/suite/metadataActions.ts

```diff
@@ -643,11 +646,6 @@ const encryptAndSaveMetadata =
             getProviderInstance({ clientId: provider.clientId, dataType: 'labels' }),
         );
 
-        if (!providerInstance) {
```

Comment:
With strict mapping this is no longer possible and this default handler can be removed.